src/cljs/replumb/load.cljs

```diff
@@ -68,32 +111,29 @@
   tries to load the unevaluated ones.
   This function does not check whether parameters are nil, please do it in
   the caller."
-  [verbose? file-names read-file-fn! load-fn-cb cached-file-names]
+  [verbose? file-names read-file-fn! load-fn-cb cached-file-names name]
```

Comment:
Ah ok sorry, it is not the same kind of opts...maybe to avoid confusion I would rename the below in something different...